netket/experimental/sampler/metropolis_pt.py

```diff
@@ -129,8 +129,19 @@ class MetropolisPtSampler(MetropolisSampler):
 
     The total number of chains evolved is :code:`n_chains * n_replicas`.
     """
+    sorted_betas: jax.Array
+    """
+    The sorted values of the temperatures for each _physical_ markov chain.
+    The first value should be β = 1 and is the _physical_ temperature.
+    """
```

Comment:
Make this internal, and add a second one for the distribution
```suggestion
    _sorted_betas: jax.Array
    """
    An internal cache for the user-specified betas, sorted.
    """
    
    _beta_distribution: str = struct.field(pytree_node=False, default="lin")
```